test/games_collection_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/games_collection'
+require './lib/game'
+require 'csv'
+
+class GamesCollectionTest < Minitest::Test
```

Comment:
This testing may also need to be modified, pending the design changes to our CSV read.